pennylane_lightning/core/src/algorithms/tests/Test_AdjointJacobian.cpp

```diff
@@ -41,7 +41,7 @@ using namespace Pennylane::LightningQubit::Observables;
 
 #elif _ENABLE_PLKOKKOS == 1
 constexpr bool BACKEND_FOUND = true;
-constexpr bool SUPPORTS_CTRL = false;
+constexpr bool SUPPORTS_CTRL = true;
```

Comment:
I don't think we need to check for this support, right?

Reply:
Indeed, this has now been removed :)